### DISCUSSION TITLE
Cookies for auth?

### DISCUSSION THREAD

**Discussion by jm61** [spinspire/pocketbase-sveltekit-starter#3] [Q&A]
Hello there, I'm experimenting with your frontend, my backend is standard pb running in localhost. No problem so far.
Just asking about auth, you are using local storage but not cookies, how server side can access this "pocketbase_auth" token without cookie?

**Comment by jkdoshi** [comment]
I don't know how to answer you question at this moment. But I'll figure out the answer and post back here shortly.
